test/support/runPlaywrightTests.js

```diff
@@ -85,7 +85,7 @@ const runTests = async (browserType) => {
     caughtError = error;
   }
 
-  mochaWebServerProcess.kill();
+  mochaServer.kill();
```

Comment:
I think it should be `mochaServer.close()`? To call this method https://github.com/ably/ably-js/pull/1652/files#diff-a020bd360991ca253146ff21e077ceeec55ff07e573e0a679b385c09b4a02f3fR58